pod.go

```diff
@@ -774,6 +776,45 @@ func (p *Pod) startVM(netNsPath string) error {
 	return nil
 }
 
+func (p *Pod) createProxyProcess() error {
+	pid, uri, err := p.proxy.start(*p)
+	if err != nil {
+		return err
+	}
+
+	// save state
+	p.state.URL = uri
+	if err := p.setPodState(p.state); err != nil {
+		return err
+	}
+
+	p.proxyProcess = newProcess("", pid)
+
+	p.Logger().WithField("proxy-pid", pid).Debug("proxy started")
```

Comment:
I'd add the pod ID as part of the log here.

Reply:
Actually, `Logger()` will already add that for us ;)